src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1327,8 +1327,6 @@ internal void GetUnaliasedReferencedAssemblies(ArrayBuilder<AssemblySymbol> asse
 
             int length = referenceManager.ReferencedAssemblies.Length;
 
-            assemblies.EnsureCapacity(assemblies.Count + length);
```

Comment:
>assemblies.EnsureCapacity(assemblies.Count + length); [](http://example.com/codeflow?start=12&length=53)

It is not obvious why this statement is removed. Do we have data proving that the removal has a noticeable positive impact? I would think that this statement was allowing us to reduce amount of times the builder was reallocated. Letting it grow on demand inside the loop might result in multiple reallocations. Assembly references with aliases are not very common, therefore, the used capacity should be pretty accurate.